java/README.md

```diff
@@ -3,13 +3,38 @@
 1. Install Maven and Java, this will depend on your OS
 1. Download the [Android SDK](https://developer.android.com/sdk/index.html)
 1. Install Android SDK Platforms 3, 10 and 19, and update the "Android SDK Tools" and "Android SDK Platform-tools"
-1. Compile the Android and Java Meterpreter, which deploys to the ../metasploit-frameworks folder
+1. Compile the Android and Java Meterpreter, which deploys to the `../metasploit-frameworks` folder
 ```
 mvn package -Dandroid.sdk.path=/path/to/android-sdk -Dandroid.release=true -P deploy
 ```
 Next time you run `msfconsole`, you should see: `WARNING: Local files may be incompatible with the Metasploit Framework`.
 This means that msfconsole is now using your newly built version of the Java and Android Meterpreter :)
 
+## Building on Docker
+
+Ensure that both the `metasploit-payloads` and `metasploit-framework` folders co-exist beside eachother:
+```
+$ ls working_directory
+metasploit-framework
+metasploit-payloads
+```
+
+First build the Docker image:
+```
+cd working_directory/metasploit-payloads/docker
+docker build -t metasploit_payloads_builder .
+```
```

Comment:
Should we add or `docker pull rapid7/msf-ubuntu-x64-meterpreter:latest`?

Reply:
I agree, maybe pull the existing image might be simpler/quicker